tests/aws/services/cloudformation/v2/test_change_set_ref.py

```diff
@@ -309,3 +309,37 @@ def test_immutable_property_update_causes_resource_replacement(
             }
         }
         capture_update_process(snapshot, template_1, template_2)
+
+    @markers.aws.validated
+    def test_supported_pseudo_parameter(
+        self,
+        snapshot,
+        capture_update_process,
+    ):
+        topic_name = f"topic-name-1-{long_uid()}"
+        snapshot.add_transformer(RegexTransformer(topic_name, "topic-name"))
+        test_name = f"test-name-{long_uid()}"
+        snapshot.add_transformer(RegexTransformer(test_name, "test-name"))
+        template_1 = {
+            "Resources": {
+                "Topic": {"Type": "AWS::SNS::Topic", "Properties": {"TopicName": topic_name}},
+            }
+        }
+        template_2 = {
+            "Resources": {
+                "Role": {
+                    "Type": "AWS::Logs::LogGroup",
+                    "Properties": {
+                        "LogGroupName": test_name,
+                        "Tags": [
+                            {"Key": "Partition", "Value": {"Ref": "AWS::Partition"}},
+                            {"Key": "AccountId", "Value": {"Ref": "AWS::AccountId"}},
+                            {"Key": "Region", "Value": {"Ref": "AWS::Region"}},
+                            {"Key": "StackName", "Value": {"Ref": "AWS::StackName"}},
+                            {"Key": "StackId", "Value": {"Ref": "AWS::StackId"}},
```

Comment:
We don't cover urlsuffix here. I realise that by definition it will be different from AWS to LocalStack but at least it would be execute the code path